src/app/health/health.component.html

```diff
@@ -8,6 +8,7 @@
   <mat-toolbar class="primary-color font-size-1">
     <a mat-raised-button color="accent" class="margin-lr-3" i18n [routerLink]="['update']" *ngIf="isOwnUser">Update Details</a>
     <a mat-raised-button color="accent" class="margin-lr-3" i18n [routerLink]="['event', { id: userDetail._id }]">Add Examination</a>
+    <span *ngIf="userDetail.firstName === undefined" >Member login: {{userDetail.name}}</span>
```

Comment:
needs space before > removed.
also based on #5232 for own health record **myHealth** will be title so **Member login** is not required